packages/providers/src/transaction-request/transaction-request.ts

```diff
@@ -35,6 +33,17 @@ import { witnessify } from './witness';
 
 export { TransactionType };
 
+// We can't import this from `@fuel-ts/script` because it causes
+// cyclic dependency errors so we duplicate it here.
+export const returnZeroScript: AbstractScript<void> = {
+  /*
+    Opcode::RET(REG_ZERO)
+    Opcode::NOOP
+  */
+  bytes: arrayify('0x24000000'),
```

Comment:
Hmmm. The encoding of instructions may (will) change in the future. Is there some way to make this more maintainable? Doesn't have to be in this PR.

Aside: `0x0000` isn't a `NOOP` IIRC, but rather an invalid instruction. Doesn't actually matter since it's unreachable.

Reply:
Yeah this is why we are moving from these to the new script package and actually writing Sway. This is the only instance that was left.

Will leave a TODO for the issue: https://github.com/FuelLabs/fuels-ts/issues/281